apollo-runtime/src/androidTest/java/com/apollographql/android/cache/normalized/sql/SqlStoreTest.java

```diff
@@ -16,15 +17,15 @@
 public class SqlStoreTest {
 
   public static final String KEY = "key";
-  public static final String FIELDS = "fields";
+  public static final String FIELDS = "{\"fieldKey\": \"value\"}";
   public static final String IN_MEMORY_DB = null; //null means db is memory only
   private SqlStore sqlStore;
 
   @Before
   public void setUp() {
     ApolloSqlHelper apolloSqlHelper = ApolloSqlHelper.create(InstrumentationRegistry.getTargetContext(),
         IN_MEMORY_DB);
-    sqlStore = SqlStore.create(apolloSqlHelper, FieldsAdapter.create(new Moshi.Builder().build()));
+    sqlStore = SqlStore.create(apolloSqlHelper, FieldsAdapter.create());
```

Comment:
I might be missing something of how Moshi works, but it seems dangerous to register our BigDecimal converter on a global Moshi instance, in case there is an existing converter. (Currently, it seems we do not allow a user to pass in a Moshi instance to Apollo anyway).

We could register these convertors on the Apollo's Moshi instance in ApolloClient. But that Moshi instance seems primarily concerned with serializing variable objects, and initializing cache serializers there would take some thought of how to do it properly without spaghetti dependencies in the ApolloClient builder.